content/docs/studio/selfhosted/installation/helm.md

```diff
@@ -0,0 +1,109 @@
+# Installing Studio Self-hosted via Helm
+
+## Prerequisites
+
+**Software**
```

Comment:
**Studio Images**
Access to the Studio Docker images need to be provided by the Iterative team to enable the installation.
```